SAKPaaS/src/app/loader.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Observable, ObservableInput, of } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+import { ApiConfiguration } from 'src/app/generated/api-configuration';
+
+export function load(http: HttpClient, config: ApiConfiguration): (() => Promise<boolean>) {
+  return (): Promise<boolean> => {
+    return new Promise<boolean>((resolve: (a: boolean) => void): void => {
+       http.get('./config.json')
+         .pipe(
+           map((x: any) => {
+             config.rootUrl = x.api_rootUrl;
+             resolve(true);
+           }),
+           catchError((x: { status: number }, caught: Observable<void>): ObservableInput<{}> => {
+             if (x.status !== 404) {
+               resolve(false);
+             }
+             config.rootUrl = 'https://api.happyhamster.org/v1';
```

Comment:
Muss das da explizit nochmal stehen? Sonst sehe ich das später das mögliche Problem, dass das zum Beispiel vergessen wird auf v2 zu ändern

Reply:
Wie meinst du? Das is halt er error catch, wenn die Datei nicht gefunden wird, also beim Entwickeln mit `ng serve` wird ja nicht aus der json gelesen, weil die nicht gefunden wird